### PR TITLE
feat: toast welcome message and lightbox removed from lh page

### PR DESCRIPTION
**Pull request by roger-in-kiva**
- toast welcome message and lightbox removed from lending home page

### CODE REVIEW

File: src/components/LoanFinding/WelcomeLightbox.vue [hunk 1]
```diff
@@ -1,73 +0,0 @@
...
-		<div class="tw-flex tw-justify-end">
-			<kv-button @click="$emit('close-lightbox')">
-				Let's go
-			</kv-button>
-		</div>
-	</div>
-</template>
-<script>
-
-import IconHeart from '@/assets/icons/inline/Heart.svg';
```
Comment (dyersituations) [inline src/components/LoanFinding/WelcomeLightbox.vue:59]: It looks like these 3 svgs can also be removed.

**Comment by mcstover** [comment]
What percentage of our user base is this still going to be totally new for?

**Comment by roger-in-kiva** [comment]
> What percentage of our user base is this still going to be totally new for?

@mcstover looks like product team is not having this accuracy.

**Comment by roger-in-kiva** [comment]
> What percentage of our user base is this still going to be totally new for?

@mcstover Lauren said: 

`logged out -it'd be new for anyone non-existing lender. last i checked, LBC traffic was  about 50%  new people, 50% existing lenders , but the CVR for new is much lower`

**Review by mcstover — APPROVED** [review]

**Review by dyersituations — APPROVED** [review]

**Pull request merged**